tests/integration/test_parameters_integration.py

```diff
@@ -20,15 +20,17 @@
 import pytest
 
 import numpy as np
-import tensorflow as tf
+# import tensorflow as tf
 
 import strawberryfields as sf
 from strawberryfields import ops
 from strawberryfields.parameters import Parameter
-from strawberryfields.backends.tfbackend import TFBackend
+# from strawberryfields.backends.tfbackend import TFBackend
 
+TFBackend = int # hack to avoid removing TFBackend from test
```

Comment:
I know it's a hack, but is there anything special about `int`?

Reply:
Nope. The only place `TFBackend` is being used is in `isinstance()` calls, which requires the arguments to be a `type`.